Enigmatry.Entry.CodeGeneration.Configuration/Form/Controls/FormControl.cs

```diff
@@ -49,4 +48,6 @@ public virtual void ApplyValidationConfiguration(IEnumerable<IFormlyValidationRu
     }
 
     public bool IsRequired => ValidationRules.HasRule<IsRequiredValidationRule>();
-}
\ No newline at end of file
+
+    public abstract string? DefaultValueAsString();
```

Comment:
I would remove this completely. 
Then, in the html helper switch on types that have default value (Input, Checkbox, etc.)